centreon/tests/e2e/features/Migration-scripts/02-media.feature

```diff
@@ -0,0 +1,69 @@
+@ignore
+@REQ_MON-24294
+
+Feature: Migration of medias from a source platform to a target platform
+
+  Background:
+    Given an admin user who wants to migrate medias from a source platform to a target platform
```

Comment:
I prefer to ask to be sure : It's really the admin user (right ACL), not "super admin" (`is_admin`)

Reply:
The source platform is supposed to be an OnPrem platform so it's the is_admin parameter